src/Tribe/Tickets_Handler.php

```diff
@@ -1195,6 +1195,14 @@ public function get_event_rsvp_tickets( $post = null ) {
 	 *             (allows zero but not `-1` for Unlimited). If oversold, will be corrected to zero.
 	 */
 	public function get_ticket_max_purchase( $ticket_id ) {
+
+		$cache = tribe( 'cache' );
+		$key = __METHOD__ . '-' . $ticket_id;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: 50f72273d7e70de752a1c698ee373d95</sub>